R/plotAbundance.R

```diff
@@ -18,92 +18,96 @@
 #'
 #' @param abund_values a \code{character} value defining which assay data to
 #'   use. (default: \code{abund_values = "relabundance"})
-#'   
-#' @param features a single \code{character} value defining a column from 
+#'
+#' @param features a single \code{character} value defining a column from
 #'   \code{colData} to be plotted below the abundance plot.
 #'   Continuous numeric values will be plotted as point, whereas factors and
 #'   character will be plotted as colour-code bar. (default: \code{features =
 #'   NULL})
-#'   
-#' @param order_rank_by How to order abundance value: By name (\dQuote{name}) 
+#'
+#' @param order_rank_by How to order abundance value: By name (\dQuote{name})
 #' for sorting the taxonomic labels alphabetically, by abundance (\dQuote{abund}) to
 #' sort by abundance values or by a reverse order of abundance values (\dQuote{revabund}).
-#'  
-#'   
+#'
+#'
```

Comment:
Additonal lines could be deleted, here is two lines and others have one between @params